scripts/do_release.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+set -xeuo pipefail
+
```

Comment:
Is it too much to ask to have a usage docs on these bash scripts? I tend to have to read the code base just to know which arguments I am supposed to give it.

Reply:
That's a good point...